django_project/core/base_templates/includes/right-nav.html

```diff
@@ -6,6 +8,43 @@
         </a>
     </li>
 {% endif %}
+
+        {% get_current_language as LANGUAGE_CODE %}
+        {% get_available_languages as LANGUAGES %}
+        {% get_language_info_list for LANGUAGES as languages %}
+<li class="dropdown">
+    <a href="#" class="dropdown-toggle" data-toggle="dropdown">
+        {% for language_header in languages %}
+             {% if language_header.code == LANGUAGE_CODE %}
+                 {% if language_header.code == 'en' %}
```

Comment:
Even better if you just put something like this:
```
<img src="{% static "img//language_flags/[LANGUAGE CODE].png" %}" alt="[LANGUAGE CODE]" style="height: 13pt; margin-bottom: 3pt;">
```
where [LANGUAGE CODE] is `language_header.code`.

I am not sure if it's possible in jinja, but it will have less code (no ifs), and no need to update code when we introduce new languages